### ISSUE TITLE
Error CS0117: 'UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for 'ApplicationName'  & 'Hello' (CS0117) (Player.Droid)

### ISSUE THREAD

**Issue by akhil-iiith** [UXDivers/Gorilla-Player-Support-And-Issues#52]
In Xamarin Studio, I'm getting the above titled error while doing build. I updated Xamarin.Forms, EngineIoClientDotNet, NewtonSoft.Json, SocketIoClientDotNet, WebSocket4Net official nuget packages to their latest versions respectively. And it throwed this error, before package updation it was working fine.

**Comment by jpbrocca** [comment]
Hi @akhil-iiith,

This could be potentially related to updating Xamarin Forms package on Android.
Did you update all the Android packages or just the Xamarin Forms package?

**Comment by akhil-iiith** [comment]
I only updated Xamarin.Forms package but since the error was there..I restored package to previous version in which it was running..then too the same error came and basically error is in Resource.designer.cs which is automatically generated by tool and in this file ApplicationName and Hello fields were missing its definition (I don't know how..I checked other Sample Apps provided..in that these fields were present. So probably by package updation/degradation these fields might have got afftected and now you manually cannot update this file). So I deleted Resource.designer.cs file from my project and did the build and now using Gorilla Player as earlier.
